code/modules/mining/satchel_ore_boxdm.dm

```diff
@@ -2,107 +2,96 @@
 /**********************Ore box**************************/
 
 /obj/structure/ore_box
-	icon = 'icons/obj/mining.dmi'
-	icon_state = "orebox0"
 	name = "ore box"
 	desc = "A heavy box used for storing ore."
-	density = 1
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "orebox0"
+	density = TRUE
 	var/last_update = 0
 	var/list/stored_ore = list()
 
-/obj/structure/ore_box/attackby(obj/item/W as obj, mob/user as mob)
-	if (istype(W, /obj/item/ore))
+/obj/structure/ore_box/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/ore))
 		user.remove_from_mob(W)
 		src.contents += W
-	if (istype(W, /obj/item/storage))
+	if(istype(W, /obj/item/storage))
 		var/obj/item/storage/S = W
 		S.hide_from(user)
 		for(var/obj/item/ore/O in S.contents)
 			S.remove_from_storage_deferred(O, src, user) //This will move the item to this item's contents
-
 			CHECK_TICK
 
 		S.post_remove_from_storage_deferred(loc, user)
-
 		to_chat(user, span("notice", "You empty the satchel into the box."))
 
 	update_ore_count()
-
 	return
 
 /obj/structure/ore_box/proc/update_ore_count()
-
 	stored_ore = list()
-
 	for(var/obj/item/ore/O in contents)
-
 		if(stored_ore[O.name])
 			stored_ore[O.name]++
 		else
 			stored_ore[O.name] = 1
 
 /obj/structure/ore_box/examine(mob/user)
-	to_chat(user, "That's an [src].")
-	to_chat(user, desc)
-
+	..()
```

Comment:
If you call parent here, check what the blow proc duplicates from parent and if there was a specific reason parent has not been called.

Reply:
The parent is atom/proc/examine, which has no functional difference.